docs/self-managed/orchestration-identity/configuration.md

```diff
@@ -7,51 +7,129 @@ description: "Learn about the Identity configuration options available in your O
+| `camunda.security.authorizations.enabled`         | If authorizations are enabled.                                                                    | `true`              |
+| `camunda.security.multitenancy.checks-enabled`    | Enables multi-tenancy checks. This requires the API to be protected.                              | `false`             |
+| `camunda.security.multitenancy.api-enabled`       | Enables the multi-tenancy API and UI independently from multi-tenancy checks.                     | `true`              |
+| `spring.profiles.active`                          | **Note:** This property will be deprecated as additional authentication methods become available. | `consolidated-auth` |
+
+  </TabItem>
+  <TabItem value="helm" label="Helm values">
+
+| Helm value key                                  | Description                                                                            | Default value |
+| ----------------------------------------------- | -------------------------------------------------------------------------------------- | ------------- |
+| `global.persistent.sessions.enabled`            | Stores session data in secondary storage so users stay logged in across cluster nodes. | `true`        |
+| `global.security.authentication.method`         | The authentication method to use. Options: `basic`, `oidc`, `none`.                    | `basic`       |
+| `global.security.authentication.unprotectedApi` | If the API can be used without authentication.                                         | `false`       |
+| `global.security.authorizations.enabled`        | If authorizations are enabled.                                                         | `true`        |
+| `global.security.multiTenancy.checksEnabled`    | Enables multi-tenancy checks. This requires the API to be protected.                   | `false`       |
+| `global.security.multiTenancy.apiEnabled`       | Enables the multi-tenancy API and UI independently from multi-tenancy checks.          | `true`        |
+
+  </TabItem>
+</Tabs>
+
+## Initialization
+
+The following variables are used to initialize users and mappings.
+
+<Tabs>
+  <TabItem value="init-env" label="Environment variables" default>
+
+| Environment variable                                | Description                          | Default value   |
+| --------------------------------------------------- | ------------------------------------ | --------------- |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_USERNAME` | The username of the first user.      | `demo`          |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_PASSWORD` | The password of the first user.      | `demo`          |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_NAME`     | The name of the first user.          | `Demo`          |
+| `CAMUNDA_SECURITY_INITIALIZATION_USERS[0]_EMAIL`    | The email address of the first user. | `demo@demo.com` |
```

Comment:
🔧 I am missing the roles assignments here. For example: 

`camunda.security.initialization.defaultRoles.admin.users[0]`

Reply:
Thanks @remcowesterhoud - I added the admin default roles. I also removed the default values from init as I think per default they are not there? Only in C8 Run which we dont document in this table